contributors/guide/README.md

```diff
@@ -5,100 +5,165 @@ aliases: [ "/guide" ]
 description:
   A small list of things that you should read and be familiar with before you
   get started with contributing. This includes such things as signing the
-  Contributor License Agreement, familiarizing yourself with our Code of Conduct
-  and more.
+  Contributor License Agreement, 
+  familiarizing yourself with our Code of Conduct, and more.
 ---
 
-# Welcome
+# Getting Started
+
+- [Getting Started](#getting-started)
+- [Welcome](#welcome)
+- [Contributor Guide](#contributor-guide)
+- [Resources Available](#resources-available)
+- [Prerequisites](#prerequisites)
+  - [Sign the CLA](#sign-the-CLA)
+  - [Code of Conduct](#code-of-conduct)
+  - [Setting Up Your Developer Environment](#setting-up)
+  - [Community Expectations and Roles](#community-expectations)
+- [Contributor Playground](#contributor-playground)
+  - [Contributor Workshops](#contributor-workshops)
+- [Community](#community)
+  - [Communication](#communication)
+  - [Events](#events)
+  - [Meetups](#meetups)
+  - [Mentorship](#mentorship)
+- [Advanced Topics](#advanced-topics)
+
+## Welcome
 
 Have you ever wanted to contribute to the coolest cloud technology?
-We will help you understand the organization of the Kubernetes project and direct you to the best places to get started.
-You'll be able to pick up issues, write code to fix them, and get your work reviewed and merged.
+This guide will help you understand the overall organization 
+of the Kubernetes project, 
+and direct you to the best places to get started contributing. 
+You'll be able to pick up issues, write code to fix them, 
+and get your work reviewed and merged.
 
-This document is the single source of truth for how to contribute to the code base.
-Feel free to browse the [open issues](https://github.com/kubernetes/community/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%2Fcontributor-guide) and file new ones, all feedback welcome!
+This document is the single source of truth for how to 
+contribute to the code base. Feel free to browse the  [open issues] 
+and file new ones, all feedback is welcome!
 
 ## Contributor Guide
 
 Welcome to Kubernetes! This guide is broken up into the following sections. 
-It is recommended that you follow these steps in order. 
+It is recommended that you follow these steps in order: 
 
 - [Welcome](#welcome) - this page 
-- [Prerequisites](#prerequisites) - things you need to complete before contributing
-- [Your First Contribution](first-contribution.md) - things you need to know before your first contribution
-- [Contributing](contributing.md) - the main guide with contributor information
+- [Prerequisites](#prerequisites) - tasks you need to complete before 
+  you can start contributing to Kubernetes
+- [Your First Contribution](first-contribution.md) - things you'll need to know 
+  before making your first contribution
+- [Contributing](contributing.md) - the main reference guide to contributing 
```

Comment:
I didn't catch these the other day, but these could both be ref links

Reply:
Done!